packages/create-cloudflare/src/frameworks/package.json

```diff
@@ -0,0 +1,36 @@
+{
+	"name": "frameworks_clis_info",
+	"dependencies": {
+		"@angular/cli": "^0.4.5",
+		"create-astro": "3.1.5",
+		"create-docusaurus": "2.4.1",
+		"create-hono": "0.2.6",
+		"create-next-app": "13.4.2",
+		"create-qwik": "1.2.7",
+		"create-react-app": "5.0.1",
+		"create-remix": "1.16.0",
+		"create-solid": "0.2.26",
+		"create-svelte": "4.2.0",
+		"create-vue": "3.6.4",
+		"gatsby": "5.10.0",
+		"nuxi": "3.4.2"
+	},
+	"//1": "this package.json is only used to keep track of the frameworks cli dependencies",
```

Comment:
could you move this above "dependencies" to make the comment easier to discover?

Reply:
no I don't think can 😓: https://github.com/cloudflare/workers-sdk/pull/3769#discussion_r1296118268

Reply:
moved to the description field as discussed 🙂